docs/prod-install.md

```diff
@@ -82,19 +92,19 @@ These settings include:
 - `EMAIL_*`:
   credentials for an outgoing SMTP server so Zulip can send emails
   when needed (don't forget to set `email_password` in the
-  `zulip-secrets.conf` file!).  We highly recommend reading our
-  [production email docs](prod-email.html) and following the test
-  procedure discussed there to make sure you've set up outgoing email
-  correctly, since outgoing email is the most common configuration
-  problem. You may also want to update `NOREPLY_EMAIL_ADDRESS`.
+  `zulip-secrets.conf` file!).
+
+- `NOREPLY_EMAIL_ADDRESS`:
+  You can use the default here, but you want the address to be something
+  that automatically gets filtered.
```

Comment:
I was using "filtered" to mean "bounced."  Can you fix the wording here and merge it?  The need to put wording here was an artifact of moving most of the section.